<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogUI</class>
 <widget class="QDialog" name="DialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditVAlitseSUmma">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelUUSIKATE">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tilille jäävä Kate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NAPPIJOU">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>440</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nosta</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHALUTTUSUMMA">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Haluttu summa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2TILILTA">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valitse tili miltä nostetaan</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditKATE">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>300</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUusiKate">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelOTSIKKO">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nosto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPOISTU">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sulje</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTILIESIKATTEET">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tiliesi Katteet</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditKATE2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelEUROA">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Euroa</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEUROA2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Euroa</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTilinValinta">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Valitse tili</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tili 1 'FI1234567891234567'</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tili 2 'FI9988776655443322'</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditTiliNumero2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdittiliNumero1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="lineHORIZONTLINE">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelEUROA3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Euroa</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelKAYTTOTILI">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Käyttötili</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSAASTOTILI">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Säästötili</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>31</width>
     <height>151</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelKuva">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/new/prefix1/Pictures/BankSimulLogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
